src/common/content-config-params.ts

```diff
@@ -81,6 +81,9 @@ export interface ContentConfigParams {
   // for the rate.
   fx: { currency: string; rate: number; timestamp: number } | undefined;
 
+  // The user's preferred units
+  preferredUnits: 'metric' | 'imperial';
+
```

Comment:
Again, it would be nice to sort this alphabetically. And maybe the comment should say "...preferred units for unit conversion"?